contrib/mpi-proxy-split/split_process.cpp

```diff
@@ -417,15 +403,7 @@ setLhMemRange()
   close(mapsfd);
   static MemRange_t lh_mem_range;
   if (found) {
-#if defined(CENTOS)
-    dynamicMemRangeHelper(&lh_mem_range, &TWO_GB);
-#elif !defined(USE_MANA_LH_FIXED_ADDRESS)
```

Comment:
Did you test your code on cori? I think this is for cori, please don't remove it unless you confirmed it works on cori as well.

Reply:
not yet, but i will send some tests to cori before i merge this in. i think the way it was written before. i will restore this segment.